### PR TITLE
Bump appveyor versions for numpy, pandas, python

### PR DESCRIPTION
**Pull request by mrocklin**
- [ ] Tests added / passed
- [ ] Passes `flake8 dask`

### CODE REVIEW

**Comment by mrocklin** [comment]
I'm surprised by the appveyor failure here.  Does anyone see what I'm doing wrong?

File: appveyor.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ environment:
 
   matrix:
     # Since appveyor is quite slow, we only use a single configuration
-    - PYTHON: "3.5"
+    - PYTHON: "3.6"
       ARCH: "64"
-      NUMPY: "1.12.1"
-      PANDAS: "0.19.2"
+      NUMPY: "1.15"
```
Comment (TomAugspurger) [inline appveyor.yml]: Maybe try exact versions like 1.15.2 and 0.23.4?

**Comment by mrocklin** [comment]
Tried without luck

**Comment by jakirkham** [comment]
There are not compatible `numpy` and `pandas` versions.

```
PackageNotFoundError: Package not found: '' Packages missing in current win-64 channels: 
  - numpy 1.15
  - pandas 0.23
```

ref: https://ci.appveyor.com/project/daskdev/dask/builds/19449313#L146

**Comment by mrocklin** [comment]
Yeah, I saw the log.  I was confused though, because there are clearly files available:

https://anaconda.org/anaconda/numpy/files

There are clearly files there.  I'll try specifying explicit versions again and push that so we can test both ways.

File: appveyor.yml (hunk 1 above)
Comment (mrocklin) [reply appveyor.yml]: Sadly not:

```
PackageNotFoundError: Package not found: '' Packages missing in current win-64 channels: 
  - numpy 1.15.2
  - pandas 0.23.4
```

**Comment by mrocklin** [comment]
```
PackageNotFoundError: Package not found: '' Packages missing in current win-64 channels: 
  - numpy 1.15.2
  - pandas 0.23.4
```

**Comment by TomAugspurger** [comment]
That's strange, since they're both there in https://repo.continuum.io/pkgs/main/win-64/

**Comment by mrocklin** [comment]
Yup.

**Pull request closed without merging**